distribution/std-lib/Base/src/Net/Http/Request.enso

```diff
@@ -0,0 +1,78 @@
+from Base import all
+import Base.Net.Uri
+import Base.Net.Http.Method
+
+type To_Request
+
+    type To_Request internal_request
+
+    to_request : Request
+    to_request = this.internal_request
+
+type To_Post_Data
+
+    type To_Post_Data internal_to_post_data
+
+    to_post_data : Any
+    to_post_data = this.internal_to_post_data
+
+## Create a GET request from Uri string
+
+   > Example
+     Create a GET request.
+         "http://httpbin.org/get".to_request
+Text.to_request : Request
```

Comment:
this looks unnecessary, passing the string as an argument is much cleaner and equally (or more) readable